plugin.video.themoviedb.helper/resources/lib/utils.py

```diff
@@ -62,10 +63,11 @@ def age_difference(birthday, deathday=''):
 
 
 def kodi_log(value, level=0):
+    logvalue = u'{0}{1}'.format(_addonlogname, value) if sys.version_info.major == 3 else u'{0}{1}'.format(_addonlogname, value).encode('ascii', 'ignore')
```

Comment:
xbmc.log accepts UTF-8 encoded byte strings. Why use 'ascii' encoding at all? Let it rest in peace.

Reply:
Ah, I wasn't sure. Updated to UTF-8
As a novice, I'm still trying to wrap my head around the decoding/encoding and differences between 2/3.